### ISSUE TITLE
The "data" JSON from the command listener contains only the "command" field, missing custom data from the voice script

### ISSUE THREAD

**Issue by Somnus007** [alan-ai/alan-sdk-reactnative#14]
Versions:
Device OS version: Android 9
React native version: 0.65.1
@alan-ai/alan-sdk-react-native: 1.6.0

Integrate code:

Subscribe

```
import {NativeEventEmitter, NativeModules} from 'react-native';
const {AlanManager, AlanEventEmitter} = NativeModules;
const alanEventEmitter = new NativeEventEmitter(AlanEventEmitter);
const alanSubscriber = alanEventEmitter.addListener('command', data => {
  console.log(`got command event ${JSON.stringify(data)}`);
});
```

Alan Button

`{alanKey && <AlanView projectid={alanKey} />}`

Description:

- Run on Android device, the "data" JSON from the command listener contains only the "command" field, missing custom data from the voice script
- Run on debug mode of Alan AI console, it returns full data.

**Comment by Somnus007** [comment]
Hi @snyuryev , any updates on this? It seems this issue has not been resolved in v1.8.0

**Comment by snyuryev** [comment]
Fixed with version 1.10.0
